### PR TITLE
Programmer Example notebook

### PR DESCRIPTION
**Pull request by riedgar-ms**
Add the Programmer regression notebook from @imatiach-msft . This is based on synthetic data, and closely follows the existing housing sample notebook.

### CODE REVIEW

**Review by imatiach-msft — APPROVED** [review]

**Comment by imatiach-msft** [comment]
maybe you can add a blurb of text at the top like "This notebook demonstrates a hypothetical scenario of how likely a programmer should be given access to a GPT2 model for inferencing, based on information such as their favorite programming language, preference for tabs vs spaces, OS, location and so forth."

**Comment by minthigpen** [comment]
I agree with @imatiach-msft 's comment, lets add some descriptive text to the top.

File: examples/notebooks/responsibleaidashboard-programmer-regression-model-debugging.ipynb [hunk 1]
```diff
@@ -0,0 +1,916 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "98605bcd",
+   "metadata": {},
+   "source": [
+    "# Analysis of Synthetic Data\n",
+    "\n",
+    "In this example, we will analyse a synthetic dataset which contains a regression problem.\n",
```
Comment (minthigpen) [inline examples/notebooks/responsibleaidashboard-programmer-regression-model-debugging.ipynb:10]: Replace with "This notebook demonstrates a hypothetical scenario of how likely a programmer should be given access to a GPT2 model for inferencing, based on information such as their favorite programming language, preference for tabs vs spaces, OS, location and so forth. Each programmer will be given a score between [0,10] where a score between [7,10] indicates access given to the programmer and [0,7) indicates access denied. The data was synthetically generated via the [PyPI package, Fibber.io](https://pypi.org/project/fibber/).

**Review by minthigpen — APPROVED** [review]
Please update the top level descriptor text.

File: examples/notebooks/responsibleaidashboard-programmer-regression-model-debugging.ipynb (hunk 1 above)
Comment (riedgar-ms) [reply examples/notebooks/responsibleaidashboard-programmer-regression-model-debugging.ipynb]: Updated.

**Comment by riedgar-ms** [comment]
@minthigpen @imatiach-msft I've updated the opening blurb. Just trying to straighten out all the paths when the notebook is running in the build automation

**Review by minthigpen — APPROVED** [review]

**Pull request merged**